### ISSUE TITLE
Finetuning (overfitting)  pi0Fast for Trossen WidowX_ai

### ISSUE THREAD

**Issue by perseusdg** [huggingface/lerobot#1811]
Hello, 

I have been trying to finetune pi0Fast for the Trossen WidowX_ai on a simple pick and place dataset. More specifically I am trying to get the model to overfit on a single trajectory from the dataset and it seems like the model really doesn't converge and the loss becomes noisy after a point. I was able to get  ACT and diffusion policy to overfit on the same dataset and when rolled out on the physical robot works perfectly unlike pi0Fast 

<img width="1229" height="361" alt="Image" src="https://github.com/user-attachments/assets/2c6e10c4-8862-49b8-ac21-9661dc8340cd" />

My training command is - `python3 src/lerobot/scripts/train.py --dataset.repo_id=/home/temp/Datasets/pick_322_two --policy.path=lerobot/pi0fast_base --output_dir=outputs/pi0_test --job_name=pi0_5090_actual_single_instance --wandb.enable=true --policy.repo_id=temp --policy.push_to_hub=false --batch_size=5 --steps=1000000 --save_freq 100000`

any insights on how i can get the model to overfit for pi0Fast or am i missing something in terms of the setup ? 

FYI - I had to bump up the pytorch version on my end (torchcodec , torchvision too) to get it working with my gpu(5090)

**Comment by perseusdg** [comment]
<img width="1226" height="318" alt="Image" src="https://github.com/user-attachments/assets/6856e893-e61a-40f3-991e-ad4eb3bead59" />

A small update , i tried to train the model from scratch by passing --policy.type=pi0fast instead of defining --policy.path. Is this the expected behavior or should i change something in my training pipeline ?

**Comment by imstevenpmwork** [comment]
PI0Fast has been deprecated in https://github.com/huggingface/lerobot/pull/2203
Please prioritize the use of the other PI0 family models
